src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -225,7 +226,8 @@ void FixupParametersFromDocs (XElement api)
 		AndroidDocScraper CreateDocScraper (string dir)
 		{
 			switch (DocletType) {
-			default: return new DroidDocScraper (dir);
+			default: return new DroidDoc2Scraper (dir);
+			case JavaDocletType.DroidDoc: return new DroidDocScraper (dir);
```

Comment:
It has changed the default doc scraper from the old droiddoc to the new droiddoc. Who's using DroidDoc? Only ourselves. Unless someone is really enthusiastic and take sources from AOSP to generate documentation using droiddoc from there, no one has chance to use it in their own project. Thus we can change the default behavior with no hesitate.

ClassParse task not being able to differentiate doclet type is another issue that I believe we can fix, just like javadoc-to-mdoc alraedy implements. But it is much easier to make that build task to take the right doclet type and parse docs as expected. That can be done in much less cost and keeps compatibility with former behavior.